library_generation/cli/entry_point.py

```diff
@@ -51,6 +51,15 @@ def main(ctx):
     metadata about library generation.
     """,
 )
+@click.option(
+    "--includes",
```

Comment:
`includes` is a little ambiguous to me that the user may not know what is the info we are including. For example, is this `api_shortname` or `library_name` or something else?  Just `library_names` or `target_library_names`(we are already using it in scripts) sounds more reasonable to me.

Reply:
Changed to `library_names`.